### DISCUSSION TITLE
0.8Beta3 - Single camera records multiple videos at the same time

### DISCUSSION THREAD

**Discussion by TheSleepySlee** [blakeblackshear/frigate#478] [Ideas]
Hello,

I am seeing multiple videos taken at the same time on same camera.
Sometimes there is a delay of 2 or 3 seconds between each video. But most of the recording is the same.
I think it would make sense to check is clip is already being recorded before starting a new one.

**Comment by mattheys** [comment]
Is this the same thing you are describing? https://github.com/blakeblackshear/frigate/issues/430

**Reply by TheSleepySlee** [reply]
Yes, sorry, you can close then.

**Reply by mattheys** [reply]
Well I am hoping for the same thing so the more people who mention it the more important it will be on the roadmap, should at least wait until Blake sees it before closing it.
